packages/drivers/form/src/actions.js

```diff
@@ -6,9 +6,6 @@ export default key => state => (name) => {
     setErrors: set('errors'),
     clearErrors: () => state.errors.reset(name),
     onChange: field => value => state.values.update({ [key]: name, [field]: value }),
-    remove: () => {
-      state.values.remove(name)
-      state.errors.remove(name)
-    },
+    remove: () => state.values.remove(name) && state.errors.remove(name),
```

Comment:
please no.
 - what if, one day remove returns `undefined` ?
 - this is not readable, remove should returns the last dispatch that is ok ?